OxfordBodleian/Juel-Jensen/BDLaethe33.xml

```diff
@@ -240,6 +253,12 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                                         <dim type="bottom">16</dim>
                                         <dim type="right">23</dim>
                                     </dimensions>
+                                    <note> The characters per line are 22.</note>
+                                    <ab type="pricking">Ruling and pricking are visible. Both primary and ruling pricks are visible.</ab>
+                                    <ab type="ruling" subtype="pattern">Ruling pattern: 1A-1A-1A1A/0-0/0-0/C.</ab>
+                                    <ab type="ruling"> The upper line is written above the ruling. The bottom line is written above the ruling.
+                                        The text has been written on one single column although the leaf was ruled to accommodate two columns. 
```

Comment:
This is really interesting! Why didn't the scribe use the space (parchment is expensive)? Maybe he saved for later miniatures? 
Does this pattern occurs over several leaves? (sorry,I don't have the images)
Maybe you could give the element locus for all patterns?